.reflame.config.jsonc

```diff
@@ -346,24 +331,20 @@
     },
     "@highlight-run/client": {
       "sourceDirectory": "sdk/client/src",
-      "entryPoints": {
-        "/src/listeners/first-load-listeners": "/listeners/first-load-listeners.tsx",
-        "/src/listeners/first-load-listeners.js": "/listeners/first-load-listeners.tsx",
-        "/src/listeners/network-listener/utils/utils": "/listeners/network-listener/utils/utils.ts",
-        "/src/utils/secure-id": "/utils/secure-id.ts",
-        "/src/utils/secure-id.js": "/utils/secure-id.ts",
-        "/src/utils/sessionStorage/highlightSession": "/utils/sessionStorage/highlightSession.ts",
-        "/src/utils/sessionStorage/highlightSession.js": "/utils/sessionStorage/highlightSession.ts",
-        "/src/utils/sessionStorage/sessionStorageKeys": "/utils/sessionStorage/sessionStorageKeys.ts",
-        "/src/utils/sessionStorage/sessionStorageKeys.js": "/utils/sessionStorage/sessionStorageKeys.ts",
-        "/src/utils/storage.js": "/utils/storage.ts",
+      "exports": {
+        "./src/listeners/first-load-listeners": "./listeners/first-load-listeners.tsx",
+        "./src/listeners/first-load-listeners.js": "./listeners/first-load-listeners.tsx",
+        "./src/listeners/network-listener/utils/utils": "./listeners/network-listener/utils/utils.ts",
+        "./src/utils/secure-id": "./utils/secure-id.ts",
+        "./src/utils/secure-id.js": "./utils/secure-id.ts",
+        "./src/utils/sessionStorage/highlightSession": "./utils/sessionStorage/highlightSession.ts",
+        "./src/utils/sessionStorage/highlightSession.js": "./utils/sessionStorage/highlightSession.ts",
+        "./src/utils/sessionStorage/sessionStorageKeys": "./utils/sessionStorage/sessionStorageKeys.ts",
+        "./src/utils/sessionStorage/sessionStorageKeys.js": "./utils/sessionStorage/sessionStorageKeys.ts",
+        "./src/utils/storage.js": "./utils/storage.ts",
```

Comment:
There's no exports map in the packageJson for this lib, so we still have to specify every entry point. Having an explicit allowlist allows us to take a bunch of shortcuts when resolving imports from libs for performance, but it should be pretty straightforward to allow importing from every file if this ever becomes a pain. Just let me know!